client/src/renderer/redux/reducers/document.js

```diff
@@ -1,5 +1,68 @@
 /** @format */
-
 import test from "./test.json";
+import { UPDATE_CARET_POSITION } from "../actions/types";
+
+const intialState = {
+  selection: {
+    start: 0,
+    end: 0,
+  },
+  nodes: translate(test),
+};
+
+/**
```

Comment:
Please give doc strings a one line title.

```suggestion
/**
 * Translate KDF nodes into Redux ready objects
```